src/Monolog/LogtailFormatter.php

```diff
@@ -26,7 +27,8 @@ public function format(array $record): string {
 
     public function formatBatch(array $records): string
     {
-        return parent::formatBatch(array_map('self::formatRecord', $records));
+        $normalized = array_values($this->normalize(array_map('self::formatRecord', $records)));
```

Comment:
`array_values` will always omit keys for us in case we encounter some other issue with JsonFormatter in the future.